operator/internal/common/extensions/secret_reconciliator.go

```diff
@@ -97,14 +97,19 @@ func (r *SecretReconciliator) EnsureSecret(ctx context.Context, name string, val
 	if err != nil {
 		return generateError(err, name, "new")
 	}
+	labels := commonLabels.DefaultLabels()
+	for k, v := range desiredLabels {
+		labels[k] = v
+	}
+
```

Comment:
You can remove this part
```suggestion
	labels := commonLabels.DefaultLabels()
	for k, v := range desiredLabels {
		labels[k] = v
	}

```